provisioning/vars/Debian.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Configure PHP paths and packages for PHP 5.6 and PHP 7.0.
+php_conf_paths:
+  - "/etc/php/{{ php_version }}/fpm"
+  - "/etc/php/{{ php_version }}/apache2"
+  - "/etc/php/{{ php_version }}/cli"
+php_extension_conf_paths:
+  - "/etc/php/{{ php_version }}/fpm/conf.d"
+  - "/etc/php/{{ php_version }}/apache2/conf.d"
+  - "/etc/php/{{ php_version }}/cli/conf.d"
+php_fpm_daemon: "php{{ php_version }}-fpm"
+php_fpm_conf_path: "/etc/php/{{ php_version }}/fpm"
+php_fpm_pool_conf_path: "{{ php_fpm_conf_path }}/pool.d/www.conf"
+php_mysql_package: "php{{ php_version }}-mysql"
+php_opcache_conf_filename: "10-opcache.ini"
+php56_packages:
```

Comment:
One potential problem here—is this list overridable via `config.yml`? As long as it is, I think I'm okay with this particular setup.